sgxs-tools/src/sgx_detect/tests/mod.rs

```diff
@@ -336,14 +352,28 @@ impl Print for EpcSize {
     }
 
     fn print(&self, level: usize) {
+        fn epc_size_unit(total_size: u64) -> (f64, String) {
```

Comment:
Could've just returned `&'static str` instead of `String`

Reply:
Adjusted, please restart bors if needed